### PR TITLE
Create Say command, might not work at all

### PR DESCRIPTION
**Pull request by robingall2910**

### CODE REVIEW

**Comment by Wild1145** [comment]
An issue will need to be raised before this is pulled as well.

File: src/com/superiornetworks/icarus/commands/Command_say.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.superiornetworks.icarus.modules;
+
+import com.superiornetworks.icarus.ICM_Rank;
+import com.superiornetworks.icarus.IcarusMod;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class Command_say extends JavaPlguin
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_say.java:9]: Cannot extend JavaPlugin,

File: src/com/superiornetworks/icarus/commands/Command_say.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import com.superiornetworks.icarus.ICM_Rank;
+import com.superiornetworks.icarus.IcarusMod;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+public class Command_say extends JavaPlguin 
+{
+    public say(IcarusMod plugin)
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_say.java:11]: Not needed, this is a command, not a module.

File: src/com/superiornetworks/icarus/commands/Command_say.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import org.bukkit.entity.Player;
+
+public class Command_say extends JavaPlguin 
+{
+    public say(IcarusMod plugin) 
+    {
+        super(plugin);
+    }
+    
+    public static void chat(CommandSender sender, String message)
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_say.java:16]: This is a command class, not a util class. This needs to be a boolean, not static, you need to add this command to plugin.yml, and you need to use the @CommandParameters notation above the class name thingy.

I highly suggest you look at a simple command class like Command_pie, and then use the stringbuilder from TFM to complete your code.

**Comment by Hockeyfan360** [comment]
There are a lot of errors that I found. If you find that my comments are inaccurate, please comment.

File: src/com/superiornetworks/icarus/commands/Command_say.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        for (Player p : Bukkit.getOnlinePlayers())
+        {
+            if (ICM_Rank.isRankOrHigher(p, ICM_Rank.Rank.SUPER))
+            {
+                // Some strange ass colour codes being used here...
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_say.java:22]: That really wasn't needed.

**Comment by Wild1145** [comment]
@xDestroyer217 Any updates with this?

**Comment by Hockeyfan360** [comment]
@xDestroyer217 Will not merge if issues cannot be fixed.

**Comment by Wild1145** [comment]
Closed for inactivity.

**Pull request closed without merging**